ek9000App/src/devEK9000.cpp

```diff
@@ -110,13 +110,18 @@ void PollThreadFunc(void*) {
 				/* check connection every other loop */
 				bool connected = device->VerifyConnection();
 				if (!connected && device->m_connected) {
-					LOG_WARNING(device, "%s: Link status changed to DISCONNECTED\n", device->m_name.data());
+					epicsPrintf("%s: Link status changed to DISCONNECTED\n", device->m_name.data());
```

Comment:
`LOG_WARNING` displays the function name and timestamp because it goes through `asynPrintf` with the warning mask. On second thought, keeping the timestamp would be wise, so I'll change this back to `LOG_WARNING`